sdk/textanalytics/Azure.AI.TextAnalytics/tests/MockTests/TextAnalyticsClientMockTests.cs

```diff
@@ -0,0 +1,679 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Pipeline;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.AI.TextAnalytics.Tests
+{
+    public class TextAnalyticsClientMockTests : ClientTestBase
+    {
+        private static readonly string s_endpoint = "https://contoso-textanalytics.cognitiveservices.azure.com/";
+        private static readonly string s_apiKey = "FakeapiKey";
+
+        public TextAnalyticsClientMockTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private TextAnalyticsClient CreateTestClient(HttpPipelineTransport transport)
+        {
+            var options = new TextAnalyticsClientOptions()
+            {
+                Transport = transport
+            };
+
+            var client = InstrumentClient(new TextAnalyticsClient(new Uri(s_endpoint), new AzureKeyCredential(s_apiKey), options));
+
+            return client;
+        }
+
+        [Test]
+        [Ignore ("Not implemented yet")]
```

Comment:
I could also delete all of the not implemented tests and add them as things start to work... but I prefer this way so there is less chance we forget to do them